BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+Folder.h

```diff
@@ -195,6 +195,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOXFolderItemsRequest *)folderItemsRequestWithID:(NSString *)folderID;
 
+/**
+ *  Generate a request to retrieve the items in a folder.
+ *
+ *  @param folderID Folder ID.
+ *  @param templateKey  The template key for specific metadata template
+ *  @param scope    The scope that then metadata is in
+ *
+ *  @return A request that can be customized and then executed.
+ */
+- (BOXFolderItemsRequest *)folderItemsRequestWithID:(NSString *)folderID
+                                        templateKey:(NSString *)templateKey
+                                              scope: (NSString *)scope;
```

Comment:
Lets rename this parameters so it's clear they are associated with metadata:
templateKey -> metadataTemplateKey
scope -> metadataScope